### PR TITLE
add templates and update nft hackathon template

### PR DESCRIPTION
**Pull request by mattcasey**
also adds scrollbars since we have so many now:
image

### CODE REVIEW

File: components/common/CreateSpaceForm/SelectNewSpaceTemplate.tsx [hunk 1]
```diff
@@ -20,56 +19,79 @@ type SelectNewSpaceTemplateProps = {
...
+              onClick={() => onSelect(template)}
+              label={spaceContentTemplates[template]}
+              icon={templateIcon[template]}
+            />
+          </Grid>
+        ))}
+        <Grid item>
+          <Divider flexItem sx={{ mb: 2 }} />
+          <Typography variant='caption' color='secondary' textTransform='uppercase' fontWeight='bold'>
+            Transfer from another platform
```
Comment (motechFR) [inline components/common/CreateSpaceForm/SelectNewSpaceTemplate.tsx:74]: This is a nice demarcation

**Comment by motechFR** [comment]
For the hackathon template, we have this "Page Not Found" showing up

Screenshot 2023-03-14 at 21 11 50

File: lib/templates/interfaces.ts [hunk 2]
```diff
@@ -19,7 +19,11 @@ export interface PageWithBlocks {
     card?: Block;
   };
   votes?: (Vote & { voteOptions: VoteOptions[] })[];
-  proposal?: (Proposal & { authors: ProposalAuthor[]; reviewers: ProposalReviewer[] }) | null;
+  proposal?:
+    | (Proposal & {
+        category: null | (ProposalCategory & { proposalCategoryPermissions: ProposalCategoryPermission[] });
```
Comment (motechFR) [inline lib/templates/interfaces.ts:24]: Why are we importing category permissions?

I think we should make import / exports focused on content only

File: lib/templates/importWorkspacePages.ts [hunk 3]
```diff
@@ -314,27 +308,47 @@ export async function generateImportWorkspacePages({
...
           bountyId: oldNewPageIdHashMap[node.id]
         });
       });
     } else if (node.type === 'proposal' && node.proposal) {
       // TODO: Handle cross space reviewers and authors
-      const { createdBy: proposalCreatedBy, authors, reviewers, ...proposal } = node.proposal;
+      const { category, ...proposal } = node.proposal;
+      let categoryId: string | undefined;
+      if (category) {
+        categoryId = space.proposalCategories.find((cat) => cat.title === category.title)?.id;
```
Comment (motechFR) [inline lib/templates/importWorkspacePages.ts:327]: I was wondering what would be the best way to handle this.

- Agree with matching the proposal categories by title.

- Disagree with creating new proposal categories. We could maybe do that thing of having a readonly general category and defaulting proposals to that one if no match found?

File: lib/templates/exportWorkspacePages.ts [hunk 4]
```diff
@@ -130,8 +130,11 @@ export async function exportWorkspacePages({
           id: node.proposalId
         },
         include: {
-          authors: true,
-          reviewers: true
+          category: {
+            include: {
+              proposalCategoryPermissions: true
```
Comment (motechFR) [inline lib/templates/exportWorkspacePages.ts:135]: I think we should ignore permissions in context of import/exports

Comment (mattcasey) [reply lib/templates/exportWorkspacePages.ts:135]: done

**Review by motechFR — APPROVED** [review]
Looks good 👍

**Pull request merged**